rlbot.fbs

```diff
@@ -17,6 +17,10 @@ table ReadyMessage {
   close_after_match:bool;
 }
 
+table InitComplete {
+  spawn_id:int;
+}
```

Comment:
Is this the new bot ready message? In that case, why not `ReadyMessage`?

Reply:
`ConnectionSettings` is the direct replacement of the old `ReadyMessage`, `InitComplete` is a bit more descriptive than `ReadyMessage` imo and distinguishes itself from the old `ReadyMessage` which had a different purpose

Reply:
Is this sent by bots or by scripts too?

I still think it could make sense to put an optional loadout in here if we are going to wait for this message anyway.

Reply:
It's waited by both but that won't work for hiveminds so I think repurposing `SetLloadout` is best

Reply:
How about an array of loadouts then? That's should work.

Reply:
`SetLoadout` is 2 items, spawn id and the loadout itself, so it would just be an array of `SetLoadout` items. I don't see the point, and that will just obscure the fact that it can also be done in the middle of a match (at least when state setting is enabled)

Reply:
I don't think it obscures it. Imo it highlights that one of them is not affected by the state setting toggle, declare loadout vs change loadout. But after thinking about it for a bit, I think you are right. It is a bit more elegant to use `SetLoadout` in both cases.